sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md

```diff
@@ -33,7 +33,7 @@
 ## 12.7.0-beta.4 (2021-05-12)
 - Added support for service version 2020-08-04.
 - Added support for Share and Share Snapshot Lease.
-- DataLakeLeaseClient now remembers the Lease ID after a lease change.
+- ShareLeaseClient now remembers the Lease ID after a lease change.
```

Comment:
Let me know if I can't update an older change log thing, I was just looking for something and I couldn't find it cause of this typo.